docs/extensions.rst

```diff
@@ -133,3 +133,37 @@ TLS
 ---
 
 See :doc:`specs/tls`.
+
+Early hints
```

Comment:
Let's capitalise Hints here, as that's how everyone refers to it

Reply:
Yep, done